src/main/java/com/gregtechceu/gtceu/api/item/ComponentItem.java

```diff
@@ -138,6 +146,9 @@ public InteractionResult useOn(UseOnContext context) {
                     return result;
                 }
             }
+            if (component instanceof IUIHolder<?>) {
```

Comment:
see https://discord.com/channels/701354865217110096/1089296351906504835/1449013507080060938, also counts for use()